pkg/registry/clusterrolebinding/policybased/storage.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
Hmm, I could see both the edit, and delete checks making sense here.

Might it be better separate this logic in to an admission controller? It'd give us a centralized place to debate and update these checks.